### PR TITLE
Phrygolydian to phrygian

### LINKED ISSUE #4219: Phrygian scale misspelled Phrygolydian
Phrygian scale is misspelled Phrygolydian.
Reported here: https://lmms.io/forum/viewtopic.php?f=7&t=27073

Reference:
https://en.wikipedia.org/wiki/Mode_%28music%29#Modern_modes
The other modes seem fine. Ionian scale is named 'Major'.

### PR DESCRIPTION
**Pull request by Anonymouqs**
Fixes #4219 which was a spelling error within the interface.

I would like to take a small moment to give thanks to all music theory geeks for catching this error ^_^ Espeically zonkmachine.

### CODE REVIEW

File: src/core/InstrumentFunctions.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ InstrumentFunctionNoteStacking::ChordTable::Init InstrumentFunctionNoteStacking:
 	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Neopolitan minor" ), { 0, 1, 3, 5, 7, 8, 11, -1 } },
 	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Hungarian minor" ), { 0, 2, 3, 6, 7, 8, 11, -1 } },
 	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Dorian" ), { 0, 2, 3, 5, 7, 9, 10, -1 } },
-	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Phrygolydian" ), { 0, 1, 3, 5, 7, 8, 10, -1 } },
+	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Phygrian" ), { 0, 1, 3, 5, 7, 8, 10, -1 } },
```
Comment (zonkmachine) [inline src/core/InstrumentFunctions.cpp]: 1. **Phygrian** -> **Phrygian**

2. This is actually the only change that should to be done. The locale files are updated automatically via the transifex platform so this is actually a one-line job. Please revert the locale file changes.

**Comment by zonkmachine** [comment]
> I would like to take a small moment to give thanks to all music theory geeks for catching this error ^_^ Espeically zonkmachine.

You got the geek part right, but I'm not the one who spot this. :wink:

File: src/core/InstrumentFunctions.cpp (hunk 1 above)
Comment (zonkmachine) [reply src/core/InstrumentFunctions.cpp]: Phrygian is still misspelled.

File: src/core/InstrumentFunctions.cpp [hunk 2]
```diff
@@ -569,13 +569,10 @@ void InstrumentFunctionArpeggio::loadSettings( const QDomElement & _this )
...
+	 	m_arpTimeKnob->setSyncMode(
  		( tempoSyncKnob::tempoSyncMode ) _this.attribute(
  						 "arpsyncmode" ).toInt() );
 	}*/
 
 	m_arpModeModel.loadSettings( _this, "arpmode" );
 }
-
-
-
```
Comment (zonkmachine) [inline src/core/InstrumentFunctions.cpp]: No. 4 spaces fine.

Comment (Anonymouqs) [reply src/core/InstrumentFunctions.cpp]: I'll restore them

Comment (Anonymouqs) [reply src/core/InstrumentFunctions.cpp]: What does No.4 spaces mean?

Comment (zonkmachine) [reply src/core/InstrumentFunctions.cpp]: It means I was somewhere else in the code... Sorry!

**Comment by zonkmachine** [comment]
I'll double check and then squash/merge this as one commit.

**Pull request merged**